static/app/views/settings/projectDataForwarding/index.tsx

```diff
@@ -114,33 +113,15 @@ function ProjectDataForwarding({project}: Props) {
     return <LoadingError onRetry={refetch} />;
   }
 
-  const plugins = pluginState.length ? pluginState : fetchedPlugins;
-
   const forwardingPlugins = () => {
     return plugins.filter(p => p.type === 'data-forwarding' && p.hasConfiguration);
   };
 
-  const updatePlugin = (plugin: Plugin, enabled: boolean) => {
-    const newPlugins = plugins.map(p => ({
-      ...p,
-      enabled: p.id === plugin.id ? enabled : p.enabled,
-    }));
-
-    setPluginState(newPlugins);
-  };
-
-  const onDisablePlugin = (plugin: Plugin) => updatePlugin(plugin, false);
-
   const hasAccess = hasEveryAccess(['project:write'], {organization, project});
 
   const pluginsPanel =
     plugins.length > 0 ? (
-      <PluginList
-        organization={organization}
-        project={project}
-        pluginList={forwardingPlugins()}
-        onDisablePlugin={onDisablePlugin}
-      />
+      <PluginList project={project} pluginList={forwardingPlugins()} />
```

Comment:
No longer passing a callback since `useTogglePluginMutation` optimistically updates & invalidates the cached values for plugins list & plugin details